projects/lexical_credo/lib/lexical_credo.ex

```diff
@@ -0,0 +1,102 @@
+defmodule LexicalCredo do
+  alias Lexical.Document
+  alias Lexical.Plugin.Diagnostic
+  alias Lexical.Project
+
+  use Diagnostic, name: :lexical_credo
+  require Logger
+
+  def init do
+    with {:ok, _} <- Application.ensure_all_started(:credo) do
+      :ok
+    end
+  end
+
+  def document do
+    %Document{}
+  end
+
+  def handle(%Document{} = doc) do
+    Application.ensure_all_started(:credo)
```

Comment:
Is this needed here as well as in `init`?

(Context: reviewing this PR for the sake of an experimental Mneme plugin 🙂)

Reply:
It is not

Reply:
@zachallaun What kind of plugins do you need in order to get Mneme working? I'm assuming diagnostics aren't enough, and you'll also need code actions.